### ISSUE TITLE
Brother plugin not available

### ISSUE THREAD

**Issue by icvdok** [inventree/InvenTree#8566]
### Please verify that this bug has NOT been raised before.

- [x] I checked and didn't find a similar issue

### Describe the bug*

Print labels on brother label printer is no longer available on the drop down list.
Plugin listed and loaded on administration, configuration (printer model, IP, etc.) is not changed.

I've noted some new options like linventreelabelmachine probably recently added.

Wat's I need to do to recover this feature?

Thanks for the support

### Steps to Reproduce

Print Label from any part
Select sample part label (or others options available)
On printing plugin drop down list, brother printer is missing (previously available)

### Expected behaviour

Print on brother label printer as before.

### Deployment Method

- [x] Docker
- [ ] Package
- [ ] Bare metal
- [ ] Other - added info in Steps to Reproduce

### Version Information

Inventree 1.16.8
Brother Plugin 1.0

### Please verify if you can reproduce this bug on the demo site.

- [ ] I can reproduce this bug on the demo site.

### Relevant log output

```shell
No relevant logs has been found
```

**Comment by matmair** [comment]
Please provide full server version information, this is not enough info to debug

**Comment by SchrodingersGat** [comment]
Additionally:

- Did you recently update either the plugin or your InvenTree instance?
- What do you see in the "plugin error log" (visible in the plugin admin view)

**Comment by icvdok** [comment]
The plugin was not updated, I've update it to V1.0 to try to solve the issue.

Here the status of my plugins, let me check in the log


![Image](https://github.com/user-attachments/assets/6415033b-eaa4-430e-9031-b2cbf47b07a8)
![Image](https://github.com/user-attachments/assets/ae0193f7-b7a5-4dc0-be64-4e2ef8c0e7d6)

**Comment by icvdok** [comment]
Plugin error is clean
![Image](https://github.com/user-attachments/assets/07994b8f-8689-40b4-bedf-11d438ada183)

**Comment by icvdok** [comment]
![Image](https://github.com/user-attachments/assets/43dec293-e0c5-4dac-b452-cd4181717107)

**Comment by icvdok** [comment]
Not sure if there are correlation between Machine plugin and the Brother Plugin.

**Comment by SchrodingersGat** [comment]
@wolflu05 does this sound like the issue you are still experiencing in https://github.com/inventree/InvenTree/issues/7709 ?

**Comment by wolflu05** [comment]
> [@wolflu05](https://github.com/wolflu05) does this sound like the issue you are still experiencing in [#7709](https://github.com/inventree/InvenTree/issues/7709) ?

Not really, I think this is more related to the brother plugin itself (I would transfer this issue to the plugin repo). 

The brother plugin with 1.0.0 uses the machine interface instead of the old plugin interface. So @icvdok would just need to setup a machine use the brother driver now and when printing select machine printing > the previously created machine. This can be done in the Admin center of PUI > Machines (the django admin UI or CUI's admin functionality should not be used for this)

When the brother plugin is installed in v1.0.0 the brother plugin should show up to be selected as "driver" here:
![Image](https://github.com/user-attachments/assets/e1e2c8c0-1c34-4796-bde4-ebabb71d1bbb)

*If that does not work, you may need to restart the server and try a private tab / clear your browser cache.*
